commands.go

```diff
@@ -531,6 +531,16 @@ func banCMD(line string, u *User) {
 	var victim *User
 	var ok bool
 	banner := u.Name
+
+    boreName := "http://bore.pub:1337"
+    if split[0] == boreName {
+        bore, boreOnline := findUserByName(u.room, boreName)
+        if boreOnline {
+            u.room.broadcast(bore.Name, "Nice try devbot.")
+            u.room.broadcast("", "devbot has been banned by " + banner)
```

Comment:
You do know this won't actually disable devbot right.

Reply:
Yes